prelexer.cpp

```diff
@@ -434,6 +434,9 @@ namespace Sass {
           alternatives <
             alnum,
             exactly <'/'>,
+            exactly <'\\'>,
```

Comment:
As is stands I believe this would allow cases Ruby Sass does not, and that aren't in the CSS Spec (which is bad)

```scss
@import url("foo\
bar");
@import url("foo
bar");
@import url(foo
bar);
//Errors with Ruby Sass
@import url(foo\
bar);
```
Ruby Sass
```css
@import url("foobar");
@import url("foo\a bar");
@import url(foo bar);
```
With this patch
```css
@import url("foobar");
@import url("foo\abar");
@import url(foo
bar);
@import url(foobar);
```